frontend/src/_styles/popover.scss

```
@@ -49,5 +49,7 @@
   }
 } 
 
-[data-radix-popper-content-wrapper] { transform: translate3d(47px,0,0) !important; }
+div[data-radix-popper-content-wrapper]:has(.PopoverContent.drawer-height) {
```

`:has` isn't supported in firefox : https://caniuse.com/css-has

```suggestion
div[data-radix-popper-content-wrapper] .PopoverContent.drawer-height {
```